packages/wrangler/src/cloudchamber/index.ts

```diff
@@ -54,5 +55,11 @@ export const cloudchamber = (
 		)
 		.command("registries", "Configure registries via Cloudchamber", (args) =>
 			registriesCommand(args).command(subHelp)
+		)
+		.command(
+			"curl <path>",
+			"perform curl in wrangler",
```

Comment:
Can you add some more information about what this command does.
Even from looking at the code, it is not 100% clear to me.
I think it is just a way to send an arbitrary request to the CC API?